Python/marshal.c

```diff
@@ -2003,7 +2072,7 @@ _PyCode_Hydrate(PyCodeObject *code)
     ctx->code = code;
```

Comment:
Two lines earlier, after rf.refs = ctx.refs, I'd like to see INCREF(refs) too, and DECREF() after the read_object() call (same as what I suggest in marshal_loads_impl()).

This is because in all other places rf.refs is owned by rf (see e.g. marshal_load_impl() -- no 's').